rllib/policy/eager_tf_policy.py

```diff
@@ -174,8 +174,15 @@ def compute_actions_from_input_dict(
         ) -> Tuple[TensorType, List[TensorType], Dict[str, TensorType]]:
             """Traced version of Policy.compute_actions_from_input_dict."""
 
+            # NOTE: In the new RLModule stack the sampling side is not traced with this
```

Comment:
I think we could trace the policy, and to do that we need to disable autograph. I'll try enabling autograph to confirm.